.pre-commit-config.yaml

```diff
@@ -55,8 +55,8 @@ repos:
     - id: rst-directive-colons
     - id: rst-inline-touching-normal
 
-# - repo: https://github.com/codespell-project/codespell
-#   rev: "v2.2.6"
-#   hooks:
-#   - id: codespell
-#     exclude_types: [json,yaml]
+- repo: https://github.com/codespell-project/codespell
+  rev: "v2.2.6"
+  hooks:
+  - id: codespell
+    args: ["-L", "fle,re-use,merchantibility,synching,crate,nin,infinit,te"]
```

Comment:
"merchantability" is a misspelling of "merchantability", but I suppose most of this is required for `LICENSE.md`.